docs/traffic-policy/concepts/phase-rules.mdx

```diff
@@ -0,0 +1,53 @@
+---
+sidebar_position: 2
+title: Phase Rules
+pagination_previous: traffic-policy/concepts/phases
+pagination_next: traffic-policy/concepts/expressions
+---
+
+# Phase Rules
+
+Each Phase is made up of Phase Rules. Phase Rules allow you to define how traffic is filtered and processed within a phase. To do this each rule consists of two key parts: **expressions** and **actions**.
+
+Here is an example:
+
+```yaml
+on_tcp_connect:
+  - expressions:
+      - conn.client_ip == '192.168.1.200'
+    actions:
+      - type: deny
+```
+
+## Expressions
```

Comment:
didn't we have a dedicated page for Expressions earlier?

Reply:
We still do, this just explains them briefly before diving into it.